tests/functional/modules/test_zos_blockinfile_func.py

```diff
@@ -264,6 +277,19 @@
 export PKG_CONFIG_PATH
 export PYTHON_HOME
 export _BPXK_AUTOCVT""",
+                  test_uss_block_insert_with_doublequotes="""//BPXSLEEP JOB MSGCLASS=A,MSGLEVEL=(1,1),NOTIFY=&SYSUID,REGION=0M
+//USSCMD EXEC PGM=BPXBATCH
+//STDERR  DD SYSOUT=*
+//STDOUT  DD SYSOUT=*
+//STDPARM DD *
+SH ls -la /;
+sleep 30;
+/*
+//
+# BEGIN ANSIBLE MANAGED BLOCK
```

Comment:
Not a dealbreaker, but might be better to add content that actually is JCL compatible, like instead of '#' comments use '//', and call cat command with SH similar to ls -la. If the change is not done no problem just makes more sense.

Reply:
Ok